projects/TD3D/td3d/td3d_transforms.py

```diff
@@ -0,0 +1,146 @@
+import numpy as np
```

Comment:
following other projects we name files like they will be named in main branch, so it's probably td3d_trainsforms.py -> transforms_3d.py